guides/developer/plugin-creators/overview.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: single
+title: "Overview"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+<div class="notice--danger">
+  <strong>Note:</strong> Plugins are an alpha feature that is under active development and may change.
+</div>
+
+
+# Requirements
+
+* Spinnaker v1.19.0
+* Orca branch `release-1.19.x` (for local testing)
+* IntelliJ IDEA (for local testing)
+
+# Plugin overview
+
+A plugin enables an operator to extend Spinnaker with custom functionality. Use cases include fetching credentials from a custom authorization service, adding a wait stage to a pipeline, updating a Jira ticket, and sending Echo events to third-party tools.
```

Comment:
There are 3 personas in plugins:

1. Core Contributors: Those who create extension points in the core services.
2. Developers: Those who create extensions / plugins implementing the extension points offered by services.
3. Operators: Those who install plugins into their Spinnaker installations.

I think it's very, very important that each persona gets its own variation of docs for the plugin experience, since each persona is going to have to know different things when it comes to this larger feature.

Reply:
@robzienert thanks for the great feedback. There already is a [Plugin Users Guide](https://www.spinnaker.io/guides/user/plugin-users/), which needs to be updated to state Spinnaker 1.19.4 and Halyard 1.34.

I've created a GitHub [issue](https://github.com/spinnaker/spinnaker.github.io/issues/1776) to create an Extension Point Creators Guide.